core-bundle/src/EventListener/Menu/BackendMenuListener.php

```diff
@@ -143,9 +147,11 @@ private function buildHeaderMenu(MenuEvent $event, BackendUser $user): void
 
         $submenu = $factory
             ->createItem('submenu')
-            ->setLabel($this->translator->trans('MSC.user', [], 'contao_default').' '.$user->username)
+            //->setLabel($this->translator->trans('MSC.user', [], 'contao_default').' '.$user->username)
+            ->setLabel('<button type="button">'.$this->translator->trans('MSC.user', [], 'contao_default').' '.$user->username.'</button>')
```

Comment:
The button element is still wrapped with a span:

<img width="536" alt="" src="https://user-images.githubusercontent.com/1192057/169283639-0debe7d4-803b-4d74-977d-00d6df9ac4f8.png">

Is this intended?

Reply:
you mean the class or the span? KnpMenu creates a span for all non-links, and it's the same for the hamburger menu toggle.